packages/server/src/api/routes/public/applications.ts

```diff
@@ -0,0 +1,155 @@
+import controller from "../../controllers/public/applications"
+import Endpoint from "./utils/Endpoint"
+const { nameValidator, applicationValidator } = require("../utils/validators")
+
+const read = [],
+  write = []
+
+/**
+ * @openapi
+ * /applications:
+ *   post:
+ *     summary: Create a new application.
+ *     tags:
+ *       - applications
+ *     parameters:
+ *       - $ref: '#/components/parameters/appId'
+ *     requestBody:
+ *       required: true
+ *       content:
+ *         application/json:
+ *           schema:
+ *             $ref: '#/components/schemas/application'
+ *     responses:
+ *       200:
+ *         description: Returns the created application.
+ *         content:
+ *           application/json:
+ *             schema:
+ *               $ref: '#/components/schemas/applicationOutput'
+ *             examples:
+ *               application:
+ *                 $ref: '#/components/examples/application'
+ */
+write.push(
+  new Endpoint("post", "/applications", controller.create).addMiddleware(
+    applicationValidator()
+  )
+)
+
+/**
+ * @openapi
+ * /applications/{appId}:
+ *   put:
+ *     summary: Update an existing application by its ID.
```

Comment:
We can probably update the terminology across the summary statements to be a bit more consistent. 

e.g: 
- PUT /applications/{appId} Update an existing application by its ID.
- PUT  /tables/{tableId}/rows/{rowId}  Update a single row within a specified table.
- PUT /tables/{tableId} Update the specified table.

Each have some slight differences in sentence structure. I also noticed these are a bit lengthy on the readme screenshot that rene posted, if we could shorten them all down to a consistent format like:
```
- POST    /path                Create an/a X
- PUT     /path/id             Update an/a X
- GET     /path                Retrieve all X
- GET     /path/id             Retrieve an/a X
- POST    /path/search         Search Xs
```

Reply:
Was thinking about doing this, will add a description with any other info that I'd like it to have and keep the summary small.